aaelf64-morello/aaelf64-morello.rst

```diff
@@ -1,13 +1,13 @@
 ..
-   Copyright (c) 2020, Arm Limited and its affiliates.  All rights reserved.
+   Copyright (c) 2020-2021, Arm Limited and its affiliates.  All rights reserved.
    CC-BY-SA-4.0 AND Apache-Patent-License
    See LICENSE file for details
 
-.. |release| replace:: 2020Q4
-.. |date-of-issue| replace:: 21\ :sup:`st` December 2020
-.. |copyright-date| replace:: 2020
-.. |footer| replace:: Copyright © |copyright-date|, Arm Limited and its
-                      affiliates. All rights reserved.
```

Comment:
Now `Non-Confidential` kind of trails behind, without being enclosed by a period. Perhaps `All rights reserved, Non-Confidential`?

Reply:
No.  There must be a period after All Rights Reserved.
`All Rights Reserved. Non-Confidential.`